dartagnan/src/main/antlr4/OpenCLLexer.g4

```diff
@@ -0,0 +1,15 @@
+lexer grammar OpenCLLexer;
+
+OpenCLWG                      :   'WG';
+OpenCLDEV                     :   'DEV';
```

Comment:
Where are these two used?

Reply:
They are used for thread scope declaration, but I think they should be in lower case, will update.